satellite/rewards/rewards.go

```diff
@@ -89,53 +92,58 @@ type Offer struct {
 	Type   OfferType
 }
 
-// IsDefault evaluates the default status of offers for templates.
-func (o Offer) IsDefault() bool {
-	if o.Status == Default {
-		return true
-	}
-	return false
+// IsEmpty evaluates whether or not an on offer is empty
+func (o Offer) IsEmpty() bool {
+	var emptyOffer Offer
```

Comment:
Would it be enough if we just do `o.name == ""` ?

Reply:
```
return o == "" 
```

Reply:
`offer` is not a string so that won't work. Also I originally used @VinozzZ logic in my template but it doesn't check all fields.

I think one extra line is fine if we check all fields just to be safe.